server/plan_dev.go

```diff
@@ -11,7 +14,7 @@ import (
 func (s *Server) registerPlanRoutes(g *echo.Group) {
 	g.GET("/plan", func(c echo.Context) error {
 		plan := &api.Plan{
-			Type: s.plan,
+			Type: s.subscription.Plan,
```

Comment:
So I come up with a new idea that we can get rid off the /plan route.

We can pre-generate the team/enterprise license for dev environment and store it in our code. And when we click the switch plan dev menu, it's essentially like we paste that license into the license field.

Thus we are reusing the existing update license logic (and has the side benefit of testing that route).

I assume our dev license can not be used in our prod license?

wdyt?

Reply:
It looks like the previous plan I mentioned in hub's PR, but the frontend will do some dirty work (load license from env then call API to switch subscription). If it's ok to store a dev license in our source code, I'm totally okay with this plan.

Reply:
I think that's fine as long as the dev license cannot be used in our release build.